plugins/chat/assets/javascripts/discourse/models/chat-thread-last-reply.js

```diff
@@ -0,0 +1,17 @@
+import { tracked } from "@glimmer/tracking";
+
+export default class ChatThreadLastReply {
```

Comment:
I think this will also be use for the thread indicators, which need to show the excerpt of the last message...will also explore caching of this + participant list at some point, need to think of it more

Reply:
yes I think it makes sense :+1:

Reply:
Maybe we could think of more generic name though... Something like `ChatThreadPreview` ? I understand we will show a lot of info of he last reply, but the way I see it, it's more a onebox of a thread than anything else.

Reply:
Actually yeah good point...since it will have the last reply, the reply count, the excerpt of the last reply, the participants and possibly more. Will go with `Preview` 👍